scipy/optimize/_linprog_doc.py

```diff
@@ -51,15 +51,17 @@ def _linprog_highs_doc(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     ----------
     c : 1-D array
         The coefficients of the linear objective function to be minimized.
-    A_ub : 2-D array, optional
+    A_ub : 2-D array or sparse matrix, optional
         The inequality constraint matrix. Each row of ``A_ub`` specifies the
-        coefficients of a linear inequality constraint on ``x``.
+        coefficients of a linear inequality constraint on ``x``. Converts 
+        input to dense array.
```

Comment:
```suggestion
        coefficients of a linear inequality constraint on ``x``.
```
This part was only needed in the method-specific documentation for the `simplex` and `revised simplex` methods. For the other inputs, the plan was to leave them sparse, right? But again, let's wait for @mckib2's input on this now that we see that our plan conflicts a bit with a previous decision.

Reply:
Yeah that's my bad. I just went through an added it to all of them without thinking, I meant to just keep it to highs and interior point. But I will hold off on correcting that as you suggest.